tests/karate/src/test/java/be/utils/JsonConverter.java

```diff
@@ -0,0 +1,143 @@
+package be.utils;
+import com.google.crypto.tink.subtle.Ed25519Sign;
+import com.intuit.karate.Json;
+import java.nio.charset.StandardCharsets;
+import java.security.*;
+import java.util.LinkedHashMap;
+import java.util.Map;
+import java.util.Base64;
+import com.google.crypto.tink.PublicKeySign;
+
+public class JsonConverter {
+
+  public Json fromMapToJson(Map<String,String> map){
+    return Json.of(map);
+  }
+
+  private String senderPk = "J9fBzJV70Jk5c-i3277Uq4CmeL4t53WDfUghaK0HpeM=";
+  private String senderSk = "0leCDBokllJXKXT72psnqF5UYFVRxnc1BNDShY05KHQn18HMlXvQmTlz6LfbvtSrgKZ4vi3ndYN9SCForQel4w==";
+
+  private String signature = "ONylxgHA9cbsB_lwdfbn3iyzRd4aTpJhBMnvEKhmJF_niE_pUHdmjxDXjEwFyvo5WiH1NZXWyXG27SYEpkasCA==";
+  private String messageIdForced = "";
+  /*
+    Produces the base64 variant of the json file passed as argument
+   */
+  public String convertJson(Json json){
```

Comment:
I am not convinced by the function name.

The Javadoc explains it (even if it is a Json object more than a file, but this is nitpicking) but the row function is not clear.

```suggestion
  public String convertJsonToBase64(Json json){
```